### ISSUE TITLE
Avoid re-using news data

### ISSUE THREAD

**Issue by ellessenne** [RConsortium/asa-biop-swe-wg#6]
Hi,
I noticed that the news data is included twice, by hand:

1. In `index.qmd`,
2. In `news.qmd`.

I think we can have data in a single file and then input it into the other, to reduce redundancy (and to make sure all is always in sync when the website is rebuilt).

I have a patch ready for it (it's straightforward), will create a new PR once #3 is merged to avoid conflicts.
